code/lighting/lighting.cpp

```diff
@@ -209,27 +202,19 @@ void light_add_point(const vec3d *pos, float r1, float r2, float intensity, floa
 
 	if (!Lighting_flag) return;
 
-	if(!specular){
-		spec_r = r;
-		spec_g = g;
-		spec_b = b;
-	}
-
 	light l;
-	
+
 	Num_lights++;
 	l.type = Light_Type::Point;
 	l.vec = *pos;
 	l.r = r;
 	l.g = g;
 	l.b = b;
-	l.spec_r = spec_r;
-	l.spec_g = spec_g;
-	l.spec_b = spec_b;
 
 	//configurable global tuning of light qualities
 	auto lp = lighting_profile::current();
 	l.intensity = lp->point_light_brightness.handle(intensity);
+	l.intensity = lp->overall_brightness.handle(l.intensity);
```

Comment:
Same here, you just reassign intensity. Either the first assignment can go, or they need to be combined somehow, or the second assignment is bad